### ISSUE TITLE
"Error in full_join_impl" Warning Running under: Ubuntu 15.10

### ISSUE THREAD

**Issue by ivanhanigan** [adamhsparks/getCRUCLdata#1]
Hi @adamhsparks 
Good to see you have this package.  I tested on R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 15.10

and get this warning

``` r
> devtools::install_github("adamhsparks/getCRUCLdata", build_vignettes = TRUE)
> library(getCRUCLdata)
> CRU_data <- create_CRU_df(pre = TRUE,
+                           pre_cv = TRUE,
+                           rd0 = TRUE,
+                           tmp = TRUE,
+                           dtr = TRUE,
+                           reh = TRUE,
+                           tmn = TRUE,
+                           tmx = TRUE,
+                           sunp = TRUE,
+                           frs = TRUE,
+                           wnd = TRUE,
+                           elv = TRUE)
 
Downloading requested data files.
 
  |==================================================================================================================================| 100%
Error in full_join_impl(x, y, by$x, by$y, suffix$x, suffix$y) : 
  'month' column not found in lhs, cannot join
> sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 15.10

locale:
 [1] LC_CTYPE=en_AU.UTF-8       LC_NUMERIC=C               LC_TIME=en_AU.UTF-8        LC_COLLATE=en_AU.UTF-8     LC_MONETARY=en_AU.UTF-8   
 [6] LC_MESSAGES=en_AU.UTF-8    LC_PAPER=en_AU.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_AU.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] getCRUCLdata_0.1.3

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.9     tidyr_0.6.1     digest_0.6.12   dplyr_0.5.0     withr_1.0.2     assertthat_0.1  plyr_1.8.4      R6_2.2.0       
 [9] DBI_0.5-1       git2r_0.18.0    magrittr_1.5    httr_1.2.1      lazyeval_0.2.0  curl_2.3        devtools_1.12.0 tools_3.3.1    
[17] purrr_0.2.2     memoise_1.0.0   knitr_1.15.1    tibble_1.2  
```

**Comment by adamhsparks** [comment]
Ah! Thanks Ivan.

I know what the issue is and will get this bug fixed ASAP.

**Comment by adamhsparks** [comment]
Fixed in 0.1.4 with https://github.com/adamhsparks/getCRUCLdata/commit/1c780add9ed754ad144a6b2cabdb441211bebfb8
